src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,22 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.3.0
+
+### New Feature
+
+- Git Catalog to provide predefined development environments to every developer
```

Comment:
Are we planning any documentation for this? (how to enable, how to use, etc...?). If not, I can try and take a stab at it this afternoon.

Reply:
@viddo raised this same question and I'm happy to take a first pass. Would I just open a PR in the docs repo or do we have a different workflow for that?

Reply:
You can open a PR directly if you want and comment it there. Or you can create a Notion doc and we can convert it easily later to include it in the docs. As far as I know we don't have a defined workflow for that...  do we? cc @RinkiyaKeDad

Reply:
[Here's my first draft](https://www.notion.so/okteto/DRAFT-Git-Catalog-Documentation-0ac65fce66404410bf70033734cb7287)